src/main/java/racing/controller/RacingController.java

```diff
@@ -23,14 +26,17 @@ public RacingController(RacingGameService racingGameService) {
     }
 
     @PostMapping("/plays")
-    public RacingGameResultResponse start(@RequestBody RacingGameInfoRequest request) {
-        Cars cars = CarFactory.carFactory(request.getNames());
-        Long gameId = racingGameService.createRacingGame(request.getCount());
+    public ResponseEntity<RacingGameResultResponse> playGame(@RequestBody RacingGameInfoRequest request) {
+        RequestValidator<RacingGameInfoRequest> requestValidator = new RacingGameInfoRequestValidator();
+        requestValidator.validate(request);
 
-        racingGameService.move(cars, request.getCount());
-        racingGameService.saveCarsState(gameId, cars);
+        Cars cars = CarFactory.carFactory(request.getNames());
+        Long gameId = racingGameService.saveGameByCount(request.getCount());
+        racingGameService.playGame(gameId, cars);
 
-        return getRacingGameResultResponse(cars);
+        return ResponseEntity
+                .status(HttpStatus.OK)
+                .body(getRacingGameResultResponse(cars));
```

Comment:
다음과 같이 간결하게도 사용할 수 있답니다 😊

```suggestion
        return ResponseEntity.ok(getRacingGameResultResponse(cars));
```